code/modules/mob/living/slime/say.dm

```diff
@@ -0,0 +1,29 @@
+/mob/living/slime/say(var/message)
+	message = sanitize(message)
```

Comment:
Is this really correct? Seems like it might double-encode.

Not entirely clear on the point of this override; shouldn't base mob handling deal with emotes properly?